### DISCUSSION TITLE
Gcode file naming

### DISCUSSION THREAD

**Discussion by 133501** [supermerill/SuperSlicer#2532] [Q&A] [answered]
Hi!
Can you please point to the direction where to look for customizing the generated gcode file name?

I 'stole' MK3S default printer profile by 'Detach from system preset' function. Edited everything including the Notes section where I've changed the: `PRINTER_VENDOR_RAISE` and `PRINTER_MODEL_P2P`. But those two parameters do not have an effect on generating gcode file naming which still comes out as: `stlName_Layer-height_material_MK3S_time.gcode`

I don't think it is a bug, just a bad setting on my part.

Not a super big problem, but I'm too lazy to rename by hand :)
Thank you!

**Comment by neophyl** [comment, marked as answer]
Detaching it from the Vendor doesn't actually remove the printer_model key from the ini file.  You have to manually edit it to change it.  Using Help>Show Configuration folder then go to that location and then look for the folder called Printer and then locate the .ini file for your detached saved profile.  
Open it up in a text editor and look for a key called "printer_model = MK3"  Then just change it to what you want and save.  Restart Susi and you should be good to go.

**Reply by 133501** [reply]
Ah! what a bliss! 
Much obliged :)
Thanks
